releasenotes/notes/feat-llmobs-function-decorator-automatic-io-annotation-acf41b686e39d20c.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    LLM Observability: This introduces automatic input/output annotation for task/tool/workflow/agent/retrieval spans traced by function decorators. 
```

Comment:
Thoughts on:
"task/tool/workflow/agent/retrieval spans created using decorators will have inputs and outputs be automatically annotated.
Manual annotations for input/output values override automatic annotations."

Reply:
nit: I personally like @Yun-Kim 's first sentence, but I like @lievan 's second sentence.